config/checkstyle_checks.xml

```diff
@@ -93,6 +93,11 @@
       <property name="message"
                 value="First sentence in a comment should start with a capital letter"/>
   </module>
+  <module name="RegexpMultiline">
```

Comment:
please add it (in addition) to:
https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_resources_checks.xml
https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_non_main_files_checks.xml
There will be violation over empty file that you just created , please add it to suppression like https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_resources_suppressions.xml#L70

Reply:
@romani There are total 30 violations. Should I add all of them to expression and solve them  in a separate issue or should I solve them in this issue itself ?

Reply:
yes, as you did not show them in comment, I will review them in PR, and we can fix them in separate PR if required, but most likely such files are required for testing. If some of them for specific javadoc Check - please skip whole folder (rather then by single files)

Reply:
Added the suppression. But there was one weird thing that I don't understand. I had to add suppressions to both `config/checkstyle_resources_suppressions.xml` and `config/checkstyle_non_main_files_suppressions.xml` otherwise they were generating violations.

Reply:
good point.
I am not sure why we have overlap between configs.
please remove Check instance from config/checkstyle_resources_suppressions.xml and its supporession file.

checkstyle_non_main_files_suppressions.xml - will cover all repo.

Reply:
Done. Please check